pages/apis/graphql/graphql_cookbook.md

```diff
@@ -541,6 +541,64 @@ mutation {
 }
 ```
 
+## Clusters
+
+A collection of common tasks with clusters using the GraphQL API.
+
+### List cluster queues 
+
+List the first 20 cluster queues' key and ID for the first 10 clusters in the organisation
```

Comment:
I think this could be a bit simpler/clearer. Maybe:

```suggestion
For the first 10 clusters in an organization, list the key and ID of the first 20 cluster queues.
```

What do you think?

Reply:
In the latest commit 👍, TY @mbelton-buildkite